storybookjs/storybook
# Fix vertical alignment of props table expandable

## What I did

The props table expandable text and arrow were vertically centered for Windows users but not for Mac users. This PR fix that.

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

Can be with the cra-ts-kitchen-sink

If your answer is yes to any of these, please make sure to include it in your PR.
